drivers/eks/eks_driver.go

```diff
@@ -41,24 +41,43 @@ const (
 )
 
 var amiForRegionAndVersion = map[string]map[string]string{
+	"1.15": map[string]string{
+		"us-east-2":      "ami-0da2b71057dfd9557",
+		"us-east-1":      "ami-0dc7713312a7ec987",
+		"us-west-2":      "ami-0e710550577202c55",
+		"ap-east-1":      "ami-00649306f8926416e",
+		"ap-south-1":     "ami-04655e6da8951ba94",
+		"ap-northeast-2": "ami-03ffb5acc344c1163",
+		"ap-southeast-1": "ami-088e05a9da0f9443c",
+		"ap-southeast-2": "ami-03f56833860168a9c",
+		"ap-northeast-1": "ami-0a1fe8c2cd382c5ec",
+		"ca-central-1":   "ami-0e540959ff231faf7",
+		"eu-central-1":   "ami-010f558f32c219ed7",
+		"eu-west-1":      "ami-01405dd0b035082f4",
+		"eu-west-2":      "ami-0603ab80fed1ae251",
+		"eu-west-3":      "ami-034e51e2c162589c1",
+		"eu-north-1":     "ami-05c92845df930f61e",
+		"me-south-1":     "ami-0e0fa92ee748fc593",
+		"sa-east-1":      "ami-02dfdb540a8de052a",
+	},
 	"1.14": map[string]string{
-		"us-east-2":      "ami-053250833d1030033",
-		"us-east-1":      "ami-07d6c8e62ce328a10",
-		"us-west-2":      "ami-07be7092831897fd6",
-		"ap-east-1":      "ami-0af3a70f827304d17",
-		"ap-south-1":     "ami-07e2a96e251e970bd",
-		"ap-northeast-2": "ami-0cf70ba01dfd0f782",
-		"ap-southeast-1": "ami-0d275f57a60281ccc",
-		"ap-southeast-2": "ami-0159ec8365aea1724",
-		"ap-northeast-1": "ami-0b60cbd90564dfe00",
-		"ca-central-1":   "ami-0ef56ecc6435d1f65",
-		"eu-central-1":   "ami-03fbd442f4f3aa689",
-		"eu-west-1":      "ami-02dca57ad67c7bf57",
-		"eu-west-2":      "ami-0a69fbeff04e330e9",
-		"eu-west-3":      "ami-074b0da576fa9f5c9",
-		"eu-north-1":     "ami-01feb408eb7fc7e23",
-		"me-south-1":     "ami-0fc6f1ff5cd458c95",
-		"sa-east-1":      "ami-010ffc66e06c843b2",
+		"us-east-2":      "ami-020a35f771eac7c58",
+		"us-east-1":      "ami-0d373fa5015bc43be",
+		"us-west-2":      "ami-0cd99c894d101fe9b",
+		"ap-east-1":      "ami-08c176ca7ce87ff08",
+		"ap-south-1":     "ami-0ce7ac18a662f8a8f",
+		"ap-northeast-2": "ami-036c8013bc6bd6051",
+		"ap-southeast-1": "ami-0ee513e646a4c80ad",
+		"ap-southeast-2": "ami-09faeb84db2d6bf65",
+		"ap-northeast-1": "ami-094212029d0e30f1e",
```

Comment:
same comment: can we move it above `ap-northeast-2` so it's alphabetical with AP region and matches the site?